File: fixtures/adversarial/R3/go/silent/before/parse.go
package parse

import "strings"

// Split the record on the separator the header names.
const todoLabel = "XXX: the queue shows this to whoever opens it"

// Parse splits a record into its fields.
func Parse(line string) []string {
	return strings.Split(line, ",")
}

// TodoLabel is what the queue shows.
func TodoLabel() string {
	return todoLabel
}

File: fixtures/adversarial/R3/rs/silent/before/src/parse.rs
// Split the record on the separator the header names.
const TODO_LABEL: &str = "TODO: the queue shows this to whoever opens it";

pub fn parse(line: &str) -> Vec<String> {
    line.split(',').map(str::to_string).collect()
}

pub fn todo_label() -> &'static str {
    TODO_LABEL
}
